VideoWeb/VideoWeb/ClientApp/src/app/shared/models/JudgeHearingSummary.ts

```diff
@@ -11,13 +11,15 @@ export class JudgeHearingSummary extends HearingSummary {
 
     get nonJudicialParticipantsExcludingObservers(): ParticipantSummary[] {
         const p = this.participants.filter(
-            x => (x.role === Role.Individual || x.role === Role.Representative) && x.hearingRole !== HearingRole.OBSERVER
+            x =>
+                (x.role === Role.Individual || x.role === Role.Representative || x.role === Role.QuickLinkParticipant) &&
```

Comment:
Can we change this to x.role === Role.QuickLinkObserver ? As the previous ticket VIH-8011 in QA is related to observer. I will bridge this as part of 7905 (QL Participant)

Reply:
This function is to exclude Observers and quick link observers. They are part of a separate table.